app/src/androidTest/java/com/github/h3lp3rs/h3lp/forum/ForumAnswersActivityTest.kt

```diff
@@ -61,8 +61,8 @@ class ForumAnswersActivityTest {
     fun addNewAnswerWorks() {
         val proUser = ProUser(USER_TEST_ID, USER_TEST_ID, "", "", "", "", "")
         proUsersDb.setObject(USER_TEST_ID, ProUser::class.java, proUser)
-        
-        forum.newPost("", QUESTION_TEST,isPost = false).thenAccept { post ->
+
+        forum.newPost("", QUESTION_TEST, isPost = false).thenAccept { post ->
```

Comment:
The USER_TEST_ID can be used in the author field instead of a blank string as done for the proUser name field. Even better, a USER_TEST_NAME val can be created for more coherence.

Reply:
That's not my code I believe ahaha, just the reformatting tool doing nasty stuff

Reply:
Changed it though